reports.html

```diff
@@ -2,26 +2,24 @@
 title: coala GSoC project reports
 ---
 
-<h1>{{ page.title }}</h1>
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>{{ page.title }}</title>
+        <meta charset="utf-8">
+        <link rel="stylesheet" href="../resources/css/style.css">
+    </head>
 
-<h2>2018</h2>
-
-<ul>
-  {% for report in site.reports %}
-    {% capture year %}{{ report.date | date: '%Y' }}{% endcapture %}
-    {% if year == '2018' %}
-      <li><a href="{{ report.url }}.html">{{ report.student }}</a>
-    {% endif %}
-  {% endfor %}
-</ul>
-
-<h2>2017</h2>
-
-<ul>
-  {% for report in site.reports %}
-    {% capture year %}{{ report.date | date: '%Y' }}{% endcapture %}
-    {% if year == '2017' %}
-      <li><a href="{{ report.url }}.html">{{ report.student }}</a>
-    {% endif %}
-  {% endfor %}
-</ul>
+    <body class="report">
```

Comment:
as you can see we are using two space indents.

please always retain the style that you find in a file, if it is consistent.  The maintainers of that file probably like the style they have chosen, and wont appreciate it changing for not reason.